documents/compute_fields_test.go

```diff
@@ -120,6 +121,15 @@ func Test_executeWASM(t *testing.T) {
 			attrs: getInvalidComputeFieldAttrs(t),
 		},
 
+		// exceeded timeout
```

Comment:
So technically the timeout logic will not work in some (linux?) machines? And that is the environment that the code will be deployed into. If calling "Close" on long running code causes a panic in the main thread, maybe makes sense to do not implement the timeout logic yet.

Reply:
Timeout logic works on all machines if the WASM has a forever loop or complex execution that goes beyond the 30 second timeout.

On some machines, the cgo is failing while stopping the instance when `timeout` exceeded. On a normal execution there should be no issues on any machines

Reply:
NVM, I have reverted back to perlin with a fix. We do not need Wasmer anymore :)